src/deepsparse/evaluation/integrations/llm_evaluation_harness.py

```diff
@@ -100,8 +100,10 @@ def integration_eval(
 
     results_raw = evaluator.simple_evaluate(**evaluator_input.dict())
 
+    inputs_dict = evaluator_input.dict()
+    del inputs_dict["model"]
```

Comment:
We need explicitly to drop `model` key from inputs. `evaluator_input.model` is a complex `DeepsparseLM` object that should be excluded from the inputs (its a big wrapper around `Pipeline`, so not only its structure is complex and ugly, but also the object is very difficult to serialize)